ServiceDescriptor.cs

```diff
@@ -394,6 +395,31 @@ public string Description
         }
 
         /// <summary>
+        /// Start mode of the Service
+        /// </summary>
+        public StartMode StartMode
+        {
+            get
+            {
+                var p = SingleElement("startmode", true);
+                if (p == null) return StartMode.Automatic;  // default value
+                try
+                {
+                    return (StartMode)Enum.Parse(typeof(StartMode), p, true);
+                }
+                catch
+                {
+                    Console.WriteLine("Start mode in XML must be one of the following:");

```

Comment:
Console IO from utility class is no good.
In winsw-2.0 it makes sense to replace it by a SEVERE error and post it in the console via log4net